exporter/awsemfexporter/metric_translator_test.go

```diff
@@ -942,16 +968,47 @@ func TestTranslateGroupedMetricToCWMetric(t *testing.T) {
 				},
 			},
 		},
+		{
+			"single metric w/ disable_metric_extraction",
+			&groupedMetric{
+				labels: map[string]string{
+					"label1": "value1",
+				},
+				metrics: map[string]*metricInfo{
+					"metric1": {
+						value: 1,
+						unit:  "Count",
+					},
+				},
+				metadata: cWMetricMetadata{
+					groupedMetricMetadata: groupedMetricMetadata{
+						namespace:   namespace,
+						timestampMs: timestamp,
+					},
+				},
+			},
+			nil,
+			true,
+			&cWMetrics{
```

Comment:
Ok so that's interesting. When we disable the EMF portion, we still return a cwMetrics struct with fields in it, but no measurements according to this test. Is the measurements the catalyst for turning into CloudWatch metrics? It's a little odd to me we stil return something.

Reply:
That is correct. These [fields](https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/27/files#diff-af3bf0e3a5a889da31842d592ffeecc4ef5d3e5b058af7ca500ed44e321bca0cR343) from CWMetric get unmarshalled into the json [here](https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/27/files#diff-af3bf0e3a5a889da31842d592ffeecc4ef5d3e5b058af7ca500ed44e321bca0cR420)